lighthouse-core/config/perf.json

```diff
@@ -12,7 +26,10 @@
     "time-to-interactive",
     "user-timings",
     "screenshots",
-    "critical-request-chains"
+    "critical-request-chains",
+    "unused-css-rules",
+    "dobetterweb/uses-optimized-images",
+    "dobetterweb/uses-responsive-images"
```

Comment:
should we add these audits as well:

links-blocking-first-paint
scripts-blocking-first-paint